### DISCUSSION TITLE
Datatype 'company' not supported for feature 'Get' by platform(s)

### DISCUSSION THREAD

**Discussion by jam-playter** [codatio/codat-docs#336] [Q&A]
I have a user that uses HSBC but couldn't connect properly. And when we check their data information the endpoint returns the 404 message. 

https://api.codat.io//companies/{id}/data/info

![image](https://user-images.githubusercontent.com/131862760/234539058-81c150a6-0d46-4a37-ab80-7ef9215f6e6c.png)

**Comment by jmyorston** [comment]
Hey @jam-playter 

Its best to contact our support team if you are unable to connect an integration (support@codat.io) - they will be best placed to assist you with this. 

The `company/data/info` endpoint is for accounting data only - so you should be able to retrieve the Sage 50 company info once you complete linking the accounting connection for this company.

**Reply by jam-playter** [reply]
Thank you will try to contact support for more details.
